docs/sdk/api-reference.md

```diff
@@ -3362,13 +2632,10 @@ Creates Input proto from keyword arguments.
 **Returns:**
 - Input proto message
 
-### Search._get_geo_point_proto()
+### Search._get_geo_point_proto
```

Comment:
I think this is a internal method. WDYT @sanjaychelliah ?

Reply:
As this is an internal method i have removed this from the doc